### ISSUE TITLE
openblas build might fail in a VM

### ISSUE THREAD

**Issue by korli** [haikuports/haikuports#1565]
see log http://vmpkg.haiku-os.org/master/x86_64/buildruns/80/output/builds/841.log

Should try with TARGET=GENERIC
```

getarch_2nd.c: In function 'main': 
getarch_2nd.c:12:35: error: 'SGEMM_DEFAULT_UNROLL_M' undeclared (first use in this function) 
    printf("SGEMM_UNROLL_M=%d\n", SGEMM_DEFAULT_UNROLL_M); 
                                  ^ 
getarch_2nd.c:12:35: note: each undeclared identifier is reported only once for each function it appears in 
getarch_2nd.c:13:35: error: 'SGEMM_DEFAULT_UNROLL_N' undeclared (first use in this function) 
    printf("SGEMM_UNROLL_N=%d\n", SGEMM_DEFAULT_UNROLL_N); 
                                  ^ 
getarch_2nd.c:14:35: error: 'DGEMM_DEFAULT_UNROLL_M' undeclared (first use in this function) 
    printf("DGEMM_UNROLL_M=%d\n", DGEMM_DEFAULT_UNROLL_M); 
                                  ^ 
getarch_2nd.c:15:35: error: 'DGEMM_DEFAULT_UNROLL_N' undeclared (first use in this function) 
    printf("DGEMM_UNROLL_N=%d\n", DGEMM_DEFAULT_UNROLL_N); 
                                  ^ 
getarch_2nd.c:19:35: error: 'CGEMM_DEFAULT_UNROLL_M' undeclared (first use in this function) 
    printf("CGEMM_UNROLL_M=%d\n", CGEMM_DEFAULT_UNROLL_M); 
                                  ^ 
getarch_2nd.c:20:35: error: 'CGEMM_DEFAULT_UNROLL_N' undeclared (first use in this function) 
    printf("CGEMM_UNROLL_N=%d\n", CGEMM_DEFAULT_UNROLL_N); 
                                  ^ 
getarch_2nd.c:21:35: error: 'ZGEMM_DEFAULT_UNROLL_M' undeclared (first use in this function) 
    printf("ZGEMM_UNROLL_M=%d\n", ZGEMM_DEFAULT_UNROLL_M); 
                                  ^ 
getarch_2nd.c:22:35: error: 'ZGEMM_DEFAULT_UNROLL_N' undeclared (first use in this function) 
    printf("ZGEMM_UNROLL_N=%d\n", ZGEMM_DEFAULT_UNROLL_N); 
                                  ^ 
getarch_2nd.c:69:50: error: 'SGEMM_DEFAULT_Q' undeclared (first use in this function) 
    printf("#define SLOCAL_BUFFER_SIZE\t%ld\n", (SGEMM_DEFAULT_Q * SGEMM_DEFAULT_UNROLL_N * 4 * 1 *  sizeof(float))); 
                                                 ^ 
getarch_2nd.c:70:50: error: 'DGEMM_DEFAULT_Q' undeclared (first use in this function) 
    printf("#define DLOCAL_BUFFER_SIZE\t%ld\n", (DGEMM_DEFAULT_Q * DGEMM_DEFAULT_UNROLL_N * 2 * 1 *  sizeof(double))); 
                                                 ^ 
getarch_2nd.c:71:50: error: 'CGEMM_DEFAULT_Q' undeclared (first use in this function) 
    printf("#define CLOCAL_BUFFER_SIZE\t%ld\n", (CGEMM_DEFAULT_Q * CGEMM_DEFAULT_UNROLL_N * 4 * 2 *  sizeof(float))); 
                                                 ^ 
getarch_2nd.c:72:50: error: 'ZGEMM_DEFAULT_Q' undeclared (first use in this function) 
    printf("#define ZLOCAL_BUFFER_SIZE\t%ld\n", (ZGEMM_DEFAULT_Q * ZGEMM_DEFAULT_UNROLL_N * 2 * 2 *  sizeof(double))); 
                                                 ^ 
make: *** [getarch_2nd] Error 1 
Makefile:123: *** OpenBLAS: Detecting CPU failed. Please set TARGET explicitly, e.g. make TARGET=your_cpu_target. Please read README for the detail..  Stop.

**Comment by miqlas** [comment]
We definetly need to define a minimum-CPU for sure as it could make incompatibility problems.
Maybe you can look into my ATLAS recipe:

https://github.com/haikuports/haikuports/pull/1117

> +	if [ "$targetArchitecture" = "x86_64" ]; then
>  +		ARCHITECTURE_BUILD_OPTS="-b 64 -s 2 -A x86SSE2 -V 768" # for x86_64
>  +	else
>  +		ARCHITECTURE_BUILD_OPTS="-b 32 -s 1 -A x86x87 -V 512" # for i686
>  +	fi

**Comment by korli** [comment]
Fixed in 8ec9c7c7ad3390dbaf8bb9c1e442e4f5054f33aa
